app/Http/Controllers/Setup/Controller.php

```diff
@@ -146,9 +178,10 @@ public function save_config() {
             self::write_config_value($config, strtoupper($k), session('setup.'.$k));
         }
 
-        if(self::is_heroku()) {
-            self::write_config_value($config, 'LOG_CHANNEL', 'stderr');
```

Comment:
Is there a case where you might want `LOG_CHANNEL` to be something else on Heroku? I set it here because I thought that was the only way to make logs appear in Heroku.

Reply:
It's still set, but via the deploy, which you don't want edited.

Reply:
I've changed